v8/medium/medium_plugin.py

```diff
@@ -33,7 +33,9 @@
 from nikola import utils
 from nikola.plugin_categories import Command
 
-LOGGER = utils.get_logger('Medium')
+from bs4 import BeautifulSoup
```

Comment:
Please use `lxml.html` instead. Nikola uses lxml for all its HTML parsing needs, and bs4 would be an extra unnecessary dependency.

Reply:
Ok that will be my goal for tomorrow ahah I should have checked before going straight for BeautifulSoup